recipe_scrapers/popsugar.py

```diff
@@ -0,0 +1,56 @@
+from ._abstract import AbstractScraper
+from ._utils import normalize_string, get_yields, get_minutes
+
+
+class PopSugar(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "popsugar.com"
+
+    def title(self):
+        title = self.context().find("h2", {"class": "recipe-title"}).get_text()
+        return normalize_string(title)
+
+    def total_time(self):
+        anchor = self.context().find(text="Total Time")
+        time = anchor.parent.findNext("dd").get_text()
+        return get_minutes(time)
+
+    def yields(self):
+        anchor = self.context().find(text="Yield")
+        serves = anchor.parent.findNext("dd").get_text()
+        return get_yields(serves)
+
+    def image(self):
+        return self.soup.find("article")["data-share-image"]
```

Comment:
Actually, one more minor detail I just noticed here: this could fail if the `data-share-image` attribute doesn't exist on a matching `article` element.

One possible pattern to avoid this is to use BeautifulSoup to search for elements where the attribute is present:

```python
article = self.soup.find("article", {"data-share-image": True})
if article:
    return article["data-share-image"]
```

(just an idea; there might be other, simpler ways to achieve this safety-check too)